webapp/migrations/Version20230302143507.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230302143507 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE board ADD remaining_seats INT NOT NULL');
```

Comment:
You probably need to provide a default value of 0 here, because I ran into this error:

```
labyrinth-webapp-migrations-1  |   SQLSTATE[23502]: Not null violation: 7 ERROR:  column "remaining_seats" of   
labyrinth-webapp-migrations-1  |   relation "board" contains null values 
```